src/modules/installers/react-web/actions.ts

```diff
@@ -181,25 +185,61 @@ export async function installSentry(): Promise<void> {
   await spinner.start();
 }
 
-export async function installComponent(component: i.Components): Promise<void> {
+export async function installComponent(component: string): Promise<void> {
   async function action() {
     const { projectName } = state.answers;
 
-    // Add dependencies without installing
-    const componentPath = `./prime-monorepo/components/web-components/${component}`;
-    if (existsSync(`${componentPath}/package.json`)) {
-      const { json: pkg } = await getPackageJson(
-        `${componentPath}/package.json`,
+    const installAndCopyComponent = async (
+      path: string,
+      destFolder: string,
+    ): Promise<string[] | null> => {
```

Comment:
```suggestion
    async function installAndCopyComponent(
      path: string,
      destFolder: string,
    ): Promise<string[] | null> {
```

function syntax voor consistency